src/serializer/types.js

```diff
@@ -29,8 +29,11 @@ export type SerializedBodyType =
 
 export type SerializedBody = {
   type: SerializedBodyType,
-  parentBody: void | SerializedBody,
   entries: Array<BabelNodeStatement>,
+  done: boolean,
+  declaredAbstractValues?: Map<AbstractValue, SerializedBody>,
+  parentBody?: SerializedBody,
```

Comment:
What is the reasoning behind moving from "parentBody: void | SerializedBody" to "parentBody?: SerializedBody"?

Reply:
It's more accurately reflecting reality, as the field itself might be absent after the constructor is done.